### ISSUE TITLE
During training have option of "fixed" augmentations on validation set

### ISSUE THREAD

**Issue by SteveOv** [SteveOv/ebop_maven#84]
Currently the validation dataset uses the same map function as the training dataset, which means that noise and roll are randomized. It would be good to at least have the option of fixing the augmentations so that the validation data remains unchanged across the training epochs; this may give smoother validation training curves.

**Comment by SteveOv** [comment]
Hmmm, this is rather difficult to do as things stand.

Options;
- roll/noise funcs set to None; no augmentations which means validation data much cleaner than training
- reset seed for roll/noise funcs each epoch; preferred option but need to find out if this is possible
- pre- noise/roll when writing validation set; could be quite a big change to how make_training_dataset works

**Comment by SteveOv** [comment]
For now running a test with roll/noise funcs set to None to see if having fixed validation data benefits training.

**Comment by SteveOv** [comment]
Won't fix
